tox_pipenv/plugin.py

```diff
@@ -19,17 +22,19 @@ def _init_pipenv_environ():
 
 
 def _clone_pipfile(venv):
-    root_pipfile_path = venv.session.config.toxinidir.join("Pipfile")
+    print(os.path.normpath(venv.envconfig.config.toxinidir))
+    root_pipfile_path = os.path.join(os.path.normpath(venv.envconfig.config.toxinidir), "Pipfile")
+    print(root_pipfile_path)
     # venv path may not have been created yet
     venv.path.ensure(dir=1)
 
     venv_pipfile_path = venv.path.join("Pipfile")
-    if not root_pipfile_path.exists():
+    if not os.path.exists(root_pipfile_path):
         with open(str(root_pipfile_path), "a"):
             os.utime(str(root_pipfile_path), None)
 
     if not venv_pipfile_path.check():
-        root_pipfile_path.copy(venv_pipfile_path)
+        root_pipfile_path = copy(venv_pipfile_path)
```

Comment:
This patch did not work for me,  I got the following error:

`:
 :  
File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 22, in <module>
    from .project import Project, SourceNotFound
  File "/usr/local/lib/python3.7/site-packages/pipenv/project.py", line 110, in <module>
    raise RuntimeError("Given PIPENV_PIPFILE is not found!")
RuntimeError: Given PIPENV_PIPFILE is not found!
` 
Changing line 37 to:
` shutil.copyfile(root_pipfile_path, venv_pipfile_path)`

Seemed to make things work for me, I'm not sure what the intention of the original line is